### ISSUE TITLE
Navigate between screens with Compose: Android Basics with Compose

### ISSUE THREAD

**Issue by brepowell** [google-developer-training/basic-android-kotlin-compose-training-cupcake#16]
"Navigate to another route," steps 2 and 3 are hard to follow. Please specify where these lines of code are supposed to go within the file.
![navigation_training](https://user-images.githubusercontent.com/85724630/200163989-b4e4a326-d5cf-4d09-8dc4-088b4a644b94.jpg)

**Comment by osuleymanova** [comment]
@brepowell 
Thank you for reaching out to us. We would really appreciate if next time you provide all the required details while reporting an issue, so we could help with the troubleshooting faster. 
Please refer to the screenshot below to locate the code you asked for:
![Screen Shot 2022-11-07 at 12 24 34 PM](https://user-images.githubusercontent.com/86437155/200409887-25099f68-0fc6-495a-b0e2-869cf218844c.png)
